lib/index.js

```diff
@@ -17,8 +17,15 @@ const routes = bedrock.config['session-http'].routes;
 // add routes
 bedrock.events.on('bedrock-express.configure.routes', function(app) {
   app.get(routes.session, asyncHandler(async (req, res) => {
-    // emit event with request and session and let listeners add session data
+    // the originalMaxAge is in ms and will result
+    // in an accurate expiration time even if the client's
+    // time is incorrect
     const session = {};
+    const {originalMaxAge} = req.session.cookie || {};
+    if(originalMaxAge) {
+      session.originalMaxAge = originalMaxAge;
```

Comment:
I see you have an EDIT based on our discussion -- so I'm assuming that this will change to `expires` and `ttl` instead.

Reply:
just so you know it looks like cookie.expires is not set unless maxAge is. Is that ok? I have no issue returning both.

Reply:
No, we want to avoid using `maxAge`. If the session does expire, we want to expose that to the client -- so since it looks like `expires` is `null` in that case, we need to create a default session expiration config in bedrock-express that session backing store modules can use as a computed property. Then we can use that value if `req.session.cookie.expires` is `null` (which is what we'll want most of the time to avoid the overhead of adding a `set-cookie` header to every response).

Reply:
We should remove `originalMaxAge` here and just use `ttl` below.

Reply:
Removed it here: https://github.com/digitalbazaar/bedrock-session-http/pull/2/commits/9cdd55328efa2279a526a5c917e9fce1f153656b